CONTRIBUTING.md

```diff
@@ -159,7 +159,8 @@ Profiling can be used to monitor the impact of Secretless on CPU and Memory cons
 - [Postgresql](https://www.postgresql.org/download/) to install Postgres
 
 We've provided sample instructions below for profiling the PostgreSQL handler.
-Note that if you are running through these instructions yourself, you'll want to
+
+*Note*: that if you are running through these instructions yourself, you'll want to
```

Comment:
Use the closing style tag at end of paragraph rather than the end of `Note` since you want the whole paragraph to stand out. Also, use imperative language after the `:` since it effectively starts a new sentence:
Current:
```
*Note*: that if you...
```

Should be:
```
*Note: If you are running...*
```

Reply:
fixed (: